umh.docs.umh.app/content/en/docs/Legacy/datamodel/database/historian/_historian.md

```
@@ -157,3 +157,4 @@ Follow our [Data Visualization](../../getstarted/datavisualization/) tutorial to
 3) Use your preferred tool to connect to our `umh_v2` database using the `kafkatopostgresqlv2` user and the password from above command.
 
    ![datagrip](/images/datamodel/database/datagrip.png)
+   
```

Markdownlint

Expected: 0 or 2; Actual: 3
Trailing spaces

(MD009, no-trailing-spaces)